source/release-notes/3.2.txt

```diff
@@ -670,6 +670,44 @@ addition to the numbers corresponding to the BSON types.
 :method:`db.collection.distinct()` method. For more information, see
 :method:`db.collection.explain()`.
 
+``keysExamined`` Statistic Corrected
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The :method:`explain()` method's output in ``executionStats`` or 
```

Comment:
Let's move most of this into the /source/reference/explain-results.txt page and try to have a short summary (linking to what you write in the explain-results page) for the 3.2 release notes page.